api/test/json-api.e2e-spec.ts

```diff
@@ -36,12 +80,11 @@ describe('JSON API Specs (e2e)', () => {
         stack: null,
       },
     };
+
     const response = await request(app.getHttpServer())
-      .post('/auth/sign-in')
-      .send({
-        username: 'fakeuser@example.com',
-        password: 'fakePassword',
-      });
+      .get('/api/v1/projects/fakeProject/features?q=fakeFeature')
```

Comment:
any clue why:
![image](https://user-images.githubusercontent.com/3466388/116670522-1ceba980-a9a0-11eb-9e4b-18048be2280b.png)

is logged to output?